api/src/utilities/unit-utilities.ts

```diff
@@ -542,3 +544,49 @@ export const summarizeUnits = (
   );
   return data;
 };
+
+export const convertToTitleCase = (value: string): string => {
```

Comment:
This and the functions would be a good functions to add unit tests for in `unit-utilities.spec.ts`